chapters/request_size.rst

```diff
@@ -3,15 +3,15 @@ Request Size
 =================
 
 ``request_size`` is the parameter that we could set for the Flow.
-It defines the number of data sent into parallel ``Peas``.
+It defines the number of ``Document`` a single ``Request`` will contain, so every ``Pea`` will receive requests containing this amount of documents.
```

Comment:
I would remove the word every

Reply:
Let's say we have 100 Documents, 10 crafters and request_size=10, this means that we will generate 10 requests each contains 10 docs and will be sent to crafters via load balancing or round robin? Is that correct?

Reply:
Like the free Pea will receive the request according to our SchedulerType

Reply:
yes, so if polling is not all, not every Pea will see all these Documents